requirements/optional.in

```diff
@@ -0,0 +1,3 @@
+# Optional dependencies go here
+
+newrelic==2.98.0.81
```

Comment:
Is there a reason this can't just go in `production.in`?

Reply:
It looks like this has been configured to install for production by default in the meantime for other reasons: https://github.com/edx/edx-analytics-data-api/pull/192 .  I'm ok either way, but it may make sense to just collapse them together at this point.  Note that we include `newrelic` in the core requirements for edx-platform: https://github.com/edx/edx-platform/blob/master/requirements/edx/base.in#L104

Reply:
Merged `optional` in `production`.